RUFAS/output_manager.py

```diff
@@ -547,10 +570,11 @@ def save_variables(self, save_path: str, dir_path: str,
                     "function": self.save_variables.__name__,
                     }
         self.add_log("exclude_info_maps", f"exclude_info_maps flag set to {exclude_info_maps}", info_map)
-        list_of_filter_files = self._list_txt_file_names_in_dir(dir_path)
+        gg = GraphGenerator()
```

Comment:
avoid using unpronounceable and vague names.
```suggestion
        graph_generator = GraphGenerator()
```